libsolidity/AST.h

```diff
@@ -415,6 +416,7 @@ class FunctionDefinition: public Declaration, public VariableScope, public Docum
 			getVisibility() >= Visibility::Internal;
 	}
 	virtual TypePointer getType(ContractDefinition const*) const override;
+	virtual bool isPartOfExternalInterface() const override { return isPublic() && m_isConstructor && getName().empty(); }
```

Comment:
thanks I already find that :) will fix

On Thu, Mar 12, 2015 at 3:26 PM, chriseth <notifications@github.com> wrote:

> In libsolidity/AST.h
> <https://github.com/ethereum/cpp-ethereum/pull/1274#discussion_r26305174>:
>
> > @@ -415,6 +416,7 @@ class FunctionDefinition: public Declaration, public VariableScope, public Docum
> >  			getVisibility() >= Visibility::Internal;
> >  	}
> >  	virtual TypePointer getType(ContractDefinition const*) const override;
> > +	virtual bool isPartOfExternalInterface() const override { return isPublic() && m_isConstructor && getName().empty(); }
>
> Should be isPublic() && !m_isConstructor && !getName().empty(); (two
> negations)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ethereum/cpp-ethereum/pull/1274/files#r26305174>.
>